icicle/curves/index.cu

```diff
@@ -1,3 +1,4 @@
 #include "bls12_377/supported_operations.cu"
 #include "bls12_381/supported_operations.cu"
-#include "bn254/supported_operations.cu"
\ No newline at end of file
+#include "bn254/supported_operations.cu"
+// #include "bw6_761/supported_operations.cu"
```

Comment:
why is this commented out ?

Reply:
Cos the wrappers aren't ready yet and I figured that this line is useless for now. I'll reactivate it once wrappers are implemented